# Teleop configuration for Logitech F710 Gamepad using the x-pad configuration.
# Left stick left drive, right stick right drive
#
#         (LB)                                 (RB)
#         (LT)                                 (RT)
#       _=====_            D(  .)X            _=====_
#      / _____ \                             / _____ \
#    +.-'_____'-.---------------------------.-'_____'-.+
#   /   |     |  '.                       .'  |      |   \
#  / ___| /|\ |___ \ (back)(Lgtc)(strt)  / ___| (Y)  |___ \
# / |      |      | ;  __           __  ; |              | |
# | | <---   ---> | | (__) .       (__) | | (X)       (B)| |
# | |___   |   ___| ; MODE         VIBE ; |___       ____| /
# |\    | \|/ |    /  _     ___      _   \    | (A) |    /|
# | \   |_____|  .','" "', |___|  ,'" "', '.  |_____|  .' |
# |  '-.______.-' /       \ANALOG/       \  '-._____.-'   |
# |               |  LJ   |------|   RJ  |                |
# |              /\       /      \       /\               |
# |             /  '.___.'        '.___.'  \              |
# |            /                            \             |
#  \          /                              \           /
#   \________/                                \_________/
#
# BUTTON         Value
#   LB             4
#   RB             5
#   A              0
#   B              1
#   X              2
#   Y              3
#
#    AXIS        Value
# Left Horiz.      0
# Left Vert.       1
# Right Horiz.     2
# Right Vert.      3
# D-pad Horiz.     5
# D-pad Vert.      6

teleop:
  joy_topic: joy
  cmd_vel_topic: cmd_vel
  left_axis: 1
  right_axis: 3
joy_node:
  deadzone: 0.1
  autorepeat_rate: 20
  coalesce_interval: 0.05
  dev: /dev/input/js0